.github/workflows/build.yml

```diff
@@ -0,0 +1,44 @@
+name: Make F3DEX3 Binary File.
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install dependencies
+      run: sudo apt install cmake build-essential
+
+    - name: Get armips
+      uses: actions/checkout@v4.1.1
+      with:
+        path: armips
+        repository: Kingcom/armips
+        submodules: "true"
+
+    - name: Build armips
+      run: |
+        cd armips
+        mkdir build && cd build
+        cmake -DCMAKE_BUILD_TYPE=Release ..
+        cmake --build .
+
+    - name: Make The Binary
+      run: make -j ARMIPS=armips/build/armips
+
+      
+    - name: Upload a Build Artifact
+      uses: actions/upload-artifact@v4.3.3
+      with:
+    # A file, directory or wildcard pattern that describes what to upload
+      path: build
+    # The desired behavior if no files are found using the provided path.
+      retention-days: 0
+      compression-level: 0
```

Comment:
like I mention this can make legal issue to post F3DEX3 binairie

Reply:
the fact that you can compile f3dex3 with just a make and it has copyrighted code IN, THE, FILES, and that's fine but binary ohhhh noooo so scary, it really doesn't matter

Reply:
I just related a recent conversation about how we should include f3dex3 in hackeroot and hackersm64 in hackern64 discord. And Sauraen was thinking to include it as submodule where the binary are compiled but not included the binary file.

Reply:
It is the community's official position that reverse engineered source code, including assembly, does not fall under the same copyright as the original copyrighted binary, or is entitled to fair use protections which would not apply to just distributing the binary. Since this is a project under the HackerN64 org, it needs to follow their standards about this.

I also don't think there's much value in distributing the artifacts. It's not like a tool where we want to make it easy for people to download and get started with limited technical ability. The only people who will want the binaries here are romhack developers, and they will at the minimum have to modify their project's build system (and game engine code) to use F3DEX3, so it's not much of an extra burden for them to have to build the microcode from source. If someone wants to make a project with minimal setup, they should use HackerOoT or HackerSM64 which already have F3DEX3 built in.

The other changes besides this, which coco already moved to a separate PR, are good and I will merge them, thanks!